rdr_service/model/bq_pdr_participant_summary.py

```diff
@@ -163,6 +163,7 @@ class BQPDRParticipantSummarySchema(BQSchema):
     # RDR CABoR tracking is to surface the appropriate authored date here.  Presence of a date (vs. null/None also
     # acts as the true/false flag equivalent to RDR participant_summary.consent_for_cabor field
     cabor_authored = BQField('cabor_authored', BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    biobank_id = BQField('biobank_id', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.NULLABLE)
```

Comment:
Is it worth adding both an INTEGER field and another `biobank_id_str` STRING field in order to align with the PDR genomics tables that have the biobank id as a string?   Or would you prefer that anyone querying the data tries a SAFE_CAST when needed to join on biobank id values?

Reply:
I don't see any reason to add a string version of the biobank id here.  The genomics biobank_id has some wonky string values, but only true integer biobank_id values will link between genomics and the pdr participant summary.